server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -263,6 +263,12 @@ public class ShuffleServerConf extends RssBaseConf {
       .defaultValue(64L * 1024L * 1024L)
       .withDescription("For multistorage, the event size exceed this value, flush data  to cold storage");
 
+  public static final ConfigOption<String> MULTISTORAGE_FALLBACK_STRATEGY_CLASS = ConfigOptions
+      .key("rss.server.multistorage.fallback.strategy.class")
+      .stringType()
+      .noDefaultValue()
```

Comment:
We should choose origin behavior as default value. 
Could we add some docs for this config option?